diff --git a/pipenv/project.py b/pipenv/project.py
index 45b387c8..4f721f84 100644
--- a/pipenv/project.py
+++ b/pipenv/project.py
@@ -407,6 +407,6 @@ class Project:
             return False
         return self.s.PIPENV_VENV_IN_PROJECT or (
-            self.project_directory and Path(self.project_directory, ".venv").is_dir()
+            self.project_directory and Path(self.project_directory, "venv").is_dir()
         )

     @property
@@ -433,21 +433,21 @@ class Project:
         # If there's no project yet, set location based on config.
         if not self.project_directory:
             if self.is_venv_in_project():
-                return Path(".venv").absolute()
+                return Path("venv").absolute()
             return get_workon_home().joinpath(self.virtualenv_name)

-        dot_venv = Path(self.project_directory) / ".venv"
+        dot_venv = Path(self.project_directory) / "venv"

-        # If there's no .venv in project root or it is a folder, set location based on config.
+        # If there's no venv in project root or it is a folder, set location based on config.
         if not dot_venv.exists() or dot_venv.is_dir():
             if self.is_venv_in_project():
                 return dot_venv
             return get_workon_home().joinpath(self.virtualenv_name)

-        # Now we assume .venv in project root is a file. Use its content.
+        # Now we assume venv in project root is a file. Use its content.
         name = dot_venv.read_text().strip()

-        # If .venv file is empty, set location based on config.
+        # If venv file is empty, set location based on config.
         if not name:
             return get_workon_home().joinpath(self.virtualenv_name)
