diff --git a/build.js b/build.js
index da338c0..a5b5cc6 100644
--- a/build.js
+++ b/build.js
@@ -46,10 +46,6 @@ await Bun.build({
     ".txt": "text",
 
   },
-  // Keep most native Node.js modules external to avoid bundling issues
-  // But don't make `sharp` external, causes issues with global Bun-based installs
-  // ref: #745, #1200
-  external: ["ws", "@vscode/ripgrep"],
   features: features,
 });
